renderer/execute/types.py

```diff
@@ -122,38 +155,14 @@ def as_error_message(self):
                     f"from {self.found_type_name} to {self.best_wanted_type_name}."
                 )
 
-        def as_quick_fixes(self):
-            """Build a QuickFix that would resolve this error."""
-            if self.should_be_text:
-                prompt = f"Convert to {self.best_wanted_type_name}"
-            else:
-                prompt = (
-                    f"Convert {self.found_type_name} to {self.best_wanted_type_name}"
-                )
-            params = {"colnames": self.column_names}
-
-            if "text" in self.wanted_types:
-                module_id = "converttotext"
-            elif "number" in self.wanted_types:
-                module_id = "converttexttonumber"
-            elif "datetime" in self.wanted_types:
-                module_id = "convert-date"
-            else:
-                raise RuntimeError(f"Unhandled wanted_types: {self.wanted_types}")
-
-            return [
-                QuickFix(
-                    I18nMessage.TODO_i18n(prompt),
-                    QuickFixAction.PrependStep(module_id, params),
-                )
-            ]
-
     def __init__(self, errors: List[PromptingError.WrongColumnType]):
         super().__init__("user must change something before we render")
         self.errors = errors
 
     def __eq__(self, other):
         return isinstance(other, self.type) and other.errors == self.errors
 
-    def as_errors_with_fixes(self) -> List[Tuple[I18nMessage, List[QuickFix]]]:
-        return [(err.as_error_message(), err.as_quick_fixes()) for err in self.errors]
+    def as_render_errors(self) -> List[RenderError]:
+        return reduce(
```

Comment:
I love functional programming as much as anybody, but in the future I suggest avoiding `reduce()`. It just doesn't work very well in Python.

Yours was:

```
return reduce(
    lambda errors, error: errors + error.as_render_errors(), self.errors, []
)
```

I suggest what I consider the Pythonic approach:

```
ret = []
for error in self.errors:
    ret.extend(error.as_render_errors())
return ret
```

Mine is 1 line longer, 10-20 chars shorter, faster and more legible.

(Don't get me wrong: I _love_ `reduce()` normally. But I have yet to find a good fit for it in Python.)

Another experiment:

```
return sum([err.as_render_errors() for err in self.errors], [])
```

This is 2 lines shorter than yours, 24 chars shorter, and equivalent. Though I suspect the average Python programmer wouldn't understand it -- just like the average Python programmer wouldn't understand `reduce()`.

I'm not asking you to change anything -- your code is correct. Just an observation for next time....